components/Header/Header.js

```diff
@@ -1,12 +1,22 @@
 import { StyledHeader, StyledTitleHeader, ToggleButton } from "./StyledHeader";
 
-export default function Header({ showForm, setShowForm }) {
+export default function Header({
+  showForm,
+  setShowForm,
+  titleSize,
+  paddingSize,
+}) {
   return (
-    <StyledHeader>
-      <StyledTitleHeader>Shopping Buddy</StyledTitleHeader>
-      <ToggleButton $showForm={showForm} onClick={() => setShowForm(!showForm)}>
-        Add Item
-      </ToggleButton>
+    <StyledHeader padding={paddingSize}>
```

Comment:
the respective styled-component is expcting "$padding" so "padding" here should be changed to "$padding"